pkg/apiserver/webhooks/datavolume-validate.go

```diff
@@ -311,19 +311,23 @@ func (wh *dataVolumeValidatingWebhook) Admit(ar v1beta1.AdmissionReview) *v1beta
 				return toAdmissionResponseError(err)
 			}
 		} else {
-			dvName, ok := pvc.Annotations[controller.AnnPopulatedFor]
-			if !ok || dvName != dv.GetName() {
-				klog.Errorf("destination PVC %s/%s already exists", dv.GetNamespace(), dv.GetName())
-				var causes []metav1.StatusCause
-				causes = append(causes, metav1.StatusCause{
-					Type:    metav1.CauseTypeFieldValueDuplicate,
-					Message: fmt.Sprintf("Destination PVC already exists"),
-					Field:   k8sfield.NewPath("DataVolume").Child("Name").String(),
-				})
-				return toRejectedAdmissionResponse(causes)
-			}
+			// In the case that the PVC exist but it it is being deleted, we create the DV, otherwise the creation is denied
```

Comment:
Don't like `else if` ?
"exist but it it is" -> "exists but it is"

Reply:
thanks @arnongilboa !